tests/hcaptcha.spec.js

```diff
@@ -503,27 +503,6 @@ describe("hCaptcha", () => {
             expect(script.src).toContain("custom=true");
         });
 
-        it("emits error when script is failed", async () => {
-            const onError = jest.fn();
-
-            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
-                    onError={onError}
-                    sitekey={TEST_PROPS.sitekey}
-                    sentry={false}
-                />);
-
-            const script = document.querySelector("head > script");
-            expect(onError.mock.calls.length).toBe(0);
-
-            script.onerror(new Error('loading failed'));
-
-            // simulate microtask
-            await Promise.reject().catch(() => null)
```

Comment:
@brdlyptrs can you confirm that this microtask is expected to not work after latest `@hcaptcha/loader` update (as now it returns resolved promise or error instead of promise)